src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -37,81 +38,7 @@ public void DisplayStats()
             .AddItem("Agility", _agility, Color.Yellow)
             .AddItem("Health", _health, Color.Red)
             .AddItem("Damage", _damage, Color.Green));
-            Console.WriteLine($"Equipment:\ntotal price: {DisplayTotalPrice()}\ntotal items: {DisplayTotalItems()}\ntotal weight; {DisplayTotalWeight()} ");
```

Comment:
This fragment should stay but you just need to use methods from the `_equipment`.